boards/arm/rpi_pico/rpi_pico.yaml

```diff
@@ -11,3 +11,4 @@ toolchain:
 supported:
   - uart
   - gpio
+  - i2c
```

Comment:
Please also add i2c to this table: https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/rpi_pico/doc/index.rst#supported-features

Reply:
I forgot about the docs, I2C is added now.